app/src/main/res/layout/fragment_details.xml

```diff
@@ -22,10 +22,8 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_below="@id/txt1"
-                android:layout_marginTop="20dp"
-                android:hint="Start Time"
-
                 android:layout_alignStart="@id/end_txt"
+                android:layout_marginTop="50dp"
```

Comment:
Reason for these changes?

Reply:
yes. If I keep it 50 then only it is properly displayed